Podcast/SeriesDetailViewController.swift

```diff
@@ -121,8 +112,8 @@ class SeriesDetailViewController: ViewController, SeriesDetailHeaderViewDelegate
         self.series = series
         title = series.title
         fetchEpisodes()
-        let deadlineTime = DispatchTime.now() + .milliseconds(100)
-        DispatchQueue.main.asyncAfter(deadline: deadlineTime) {
+        
+        DispatchTime.waitFor(milliseconds: 100) {
```

Comment:
Can you do a search all and switching every place we has DispathTime.now() + x seconds to now be this function. Thanks!

Reply:
That's the only instance of DispatchTime.now() I found!